staging/src/k8s.io/cloud-provider/controllers/node/node_controller.go

```diff
@@ -295,7 +432,8 @@ func (cnc *CloudNodeController) updateNodeAddress(ctx context.Context, node *v1.
 // in a retry-if-conflict loop.
 type nodeModifier func(*v1.Node)
 
-func (cnc *CloudNodeController) UpdateCloudNode(ctx context.Context, _, newObj interface{}) {
+// UpdateCloudNode handles updating existing nodes registered with the cloud taint.
+func (cnc *CloudNodeController) UpdateCloudNode(ctx context.Context, newObj interface{}) {
```

Comment:
maybe just name this  `syncNode` now?

Reply:
Also thinking that  syncNode should receive the name of the node instead of `newObj interface{}` and in `initializeNode` we fetch the latest node from the informer cache line 478, instead of from apiserver.